pkg/acs/calacs/lib/mkspt.c

```diff
@@ -191,6 +214,27 @@ int mkNewSpt (char *in_list, char *mtype, char *output) {
         freeHdr (&header);
     }
 
+    /* If necessary, re-open the primary header to update the NEXTEND keyword */
+    if ((extnum != numSPT) || (numSPT != nimgs)) {
+
+        im = openUpdateImage (out_spt, "", 0, &header);
+        if (hstio_err()) {
+            trlopenerr (out_spt);
+            return (status = OPEN_FAILED);
+        }
+
+        if (PutKeyInt (&header, "NEXTEND", extnum, ""))
```

Comment:
I'm unsure of this, so am just going to throw it out there - is this correct? Should `NEXTEND` be updated with `extnum`?

Reply:
Yes.  The variable extnum represents the total number of SPT arrays in all the *existing* input _spt.fits files, accounting for the fact that an _spt.fits file could have multiple arrays (extensions).

Reply:
K